Automattic/wp-calypso
# Show Jetpack 2 year plans on pricing page as owned

## Proposed Changes

* Allow bi-yearly plan slugs in pricing page grid
* Add constants for Jetpack Backup Bi-yearly
* This fixes an issue where the products, when purchased with a bi-yearly plan, disappeared off the pricing page completely

## Testing Instructions

1. Use the Calypso green live link to go to the pricing page with a site in context (/pricing/{site}) (or checkout locally and go to `/pricing/{site}`)
2. With a site in context, purchase the bi-yearly plans for Jetpack Boost, Jetpack Scan, Jetpack Anti-spam, Jetpack Videopress, Jetpack Backup, Jetpack Stats, and Jetpack AI
3. Go back to the pricing page with the site in context
4. Make sure all the products still show up, correctly showing which plans are and are not active
![image](https://github.com/Automattic/wp-calypso/assets/65001528/0e840fea-7337-4728-9243-7bda7263c6f1)


## Pre-merge Checklist
